nodejs/week3/warmup/index.js

```diff
@@ -0,0 +1,130 @@
+const express = require("express");
+const app = express();
+
+//app.get("/", (req, res) => res.send("nodejs week3 homework"));
+
+app.get("/calculator/add", (req, res) => {
+  const firstParam = parseInt(req.query.firstParam);
+  const secondParam = req.query.secondParam;
```

Comment:
One of the homework requests is: There can be an infinite number of query parameters! Not only two.